Userland/Libraries/LibGL/GL/gl.h

```diff
@@ -59,6 +59,10 @@ extern "C" {
 #define GL_CW 0x0900
 #define GL_CCW 0x0901
 
+// Listing enums
+#define GL_COMPILE 0
```

Comment:
Wie should use official values for these defines. See https://www.khronos.org/registry/OpenGL/api/GLSC/1.0.1/gl.h

Reply:
The link I posted is for GLSC which I didn't even kbow was a thing... the values are the same though but please check again.